obspy/imaging/cm.py

```diff
@@ -24,42 +31,69 @@
 import numpy as np
 from matplotlib.cm import get_cmap
 from matplotlib.colors import LinearSegmentedColormap
+from matplotlib.colors import ListedColormap
 
 
-def _get_cmap(name, lut=None, reverse=False):
+def _get_cmap(file_name, lut=None, reverse=False):
```

Comment:
yes, name is the name of the colormap, which is stored by matplotlib when constructing the colormap e.g. "viridis", and the input should be really the file name now (which has changed). It's more clear like this because the API has changed as well. It might be nice to jsut search all files in data and import them as colormap. Then nobody has to look at cm.py again. I don't know how to import the names into the cm namespace though...